internal/featureflags/featureflag.go

```diff
@@ -17,23 +17,33 @@
 package featureflags
 
 import (
-	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/log"
 	"os"
 	"strconv"
 	"strings"
+
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/log"
+)
+
+type (
+	// FeatureFlagID is the environment variable name
+	FeatureFlagID string
```

Comment:
I think the best name is just `ID`.
Then on the caller side, it would be `featureFlags.ID`.
```go
	tests := []struct {
		name         string
		given        downloadConfigsOptions
		want         wantDownload
		featureFlags map[featureflags.ID]bool
	}{
```